### DISCUSSION TITLE
[ 功能请求 ]-添加对 StarRocks 数据库的支持

### DISCUSSION THREAD

**Discussion by wuliLSF** [hhyo/Archery#1975] [Ideas]
Archery使用 MySQL 的客户端可以连接到 StarRcoks 数据库，目前可以进行查询和数据字典操作，希望可以添加 SQL 审核功能，或者添加对 StarRocks 数据库的支持

**Comment by Invinciblegun** [comment]
查询的时候报如下错误
无法校验查询语句权限，请联系管理员，错误信息：list index out of range
怎么处理

**Reply by wuliLSF** [reply]
不通过 archery 查询也报错么？看看你是不是权限不足啊

**Comment by cxheny** [comment]
正常走MySQL协议 也能操作基本的查询操作。

**Reply by wuliLSF** [reply]
主要是想要sql审核的功能
